File: app/src/main/java/org/jellyfin/androidtv/startup/SelectUserActivity.java
package org.jellyfin.androidtv.startup;

import android.os.Bundle;

import androidx.fragment.app.FragmentActivity;

public class SelectUserActivity extends FragmentActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        getSupportFragmentManager()
                .beginTransaction()
                .replace(android.R.id.content, new SelectUserFragment())
                .commit();
    }
}

File: app/src/main/java/org/jellyfin/androidtv/startup/SelectUserActivity.kt
package org.jellyfin.androidtv.startup

import android.os.Bundle
import androidx.fragment.app.FragmentActivity

class SelectUserActivity : FragmentActivity() {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)

		supportFragmentManager
			.beginTransaction()
			.replace(android.R.id.content, SelectUserFragment())
			.commit()
	}
}
